File: Assignments/Assignment_6/program2.c
// Write a program which accept one number from user and check whether that number is greater than 100 or not.

#include<stdio.h>
#include<stdbool.h>

////////////////////////////////////////////////////////////////////
//
//  Function Name : ChkGreater
//  Description :   It is used to display number is greater than 100 or not.
//  Input :         Integer
//  Output :        boolean
//  Author :        Omkar Sachin Narlawar
//  Date :          26/10/2025
//
////////////////////////////////////////////////////////////////////

bool ChkGreater(int iNo)
{
    if(iNo > 100)
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    int iValue = 0;
    bool bRet = false;

    printf("Please enter number\n");
    scanf("%d",&iValue);

    bRet = ChkGreater(iValue);

    if(bRet == true)
    {
        printf("Greater");
    }
    else
    {
        printf("Smaller");
    }

    return 0;
}

// TIme Complexity : O(1)

File: Assignments/Assignment_57/program2.java
//  Write java program to accept file name from user and check whether that file is regular file or not.

import java.util.Scanner;
import java.io.*;

class program2
{
    public static void main(String A[])
    {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter file name : ");
        String name = sc.nextLine();

        File fobj = new File(name);

        if(fobj.exists() && fobj.isFile())
        {
            System.out.println("It is regular file");
        }
        else
        {
            System.out.println("It is no a regular file");
        }

        sc.close();
    }
}